packages/io-fims-common/src/domain/audit-event.ts

```diff
@@ -0,0 +1,41 @@
+import { z } from "zod";
+
+import { userMetadataSchema } from "./user-metadata.js";
+
+export const requestParamsSchema = z.object({
+  client_id: z.string().min(1),
+  redirect_uri: z.string().min(1).optional(),
```

Comment:
Can we use url here?
```suggestion
  redirect_uri: z.string().url().optional(),
```